content/manual/running/vere.md

```diff
@@ -197,6 +198,29 @@ command.
 Note the auto-dock behavior when booting new ships can be disabled with the
 `--no-dock` option.
 
+### `eval`
+
+Evaluate a hoon expression without booting a ship.
+
+The expression to evaluate is given in a string via stdin, and the result is
+pretty-printed to the terminal. Note you do not need to boot an actual ship to
+run this, the runtime can do it itself.
+
+- Undocked: `echo [expression] | urbit eval`
+- Docked: `echo [expression] | [pier]/.run eval`
+
+This will work like:
```

Comment:
Is this just against the standard subject then, or the current Dojo subject?

Reply:
no it looks like it's eth & azimuth utilties. Idk where it comes from or its purpose

```
❯ echo "." | urbit eval
loom: mapped 2048MB
lite: arvo formula 11a9e7fe
lite: core 38d4ad4d
lite: final state 38d4ad4d
eval:
< 1.mrs
  [   ethereum
    < 12.zya
      17.ayh
      34.bxr
      14.yze
      54.jwx
      77.kdc
      232.gkb
      51.qbt
      123.ppa
      46.hgz
      1.pnw
      %140
    >
      azimuth
    < 6.svq
      11.gpf
      [ < 4.kqd
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
        < 3.hqp
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
          ethereum
        < 12.zya
          17.ayh
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
        < 17.ayh
          34.bxr
          14.yze
          54.jwx
          77.kdc
          232.gkb
          51.qbt
          123.ppa
          46.hgz
          1.pnw
          %140
        >
      ]
    >
    < 17.ayh
      34.bxr
      14.yze
      54.jwx
      77.kdc
      232.gkb
      51.qbt
      123.ppa
      46.hgz
      1.pnw
      %140
    >
  ]
>
```